dev/com.ibm.ws.security.auth.data.common/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -32,3 +32,14 @@ enabled.desc=Enables the authentication data subject cache.
 
 maxSize=Maximum cache size
 maxSize.desc=Maximum number of entries supported by the authentication data subject cache.
+
+#======= KERBEROS SETTINGS =========
+# Do not translate: Kerberos, principal
+krb5Principal=Kerberos principal name
+krb5Principal.desc=The name of the principal to be used. The principal can be a simple username or a service name.
+
+# Do not translate: Kerberos, keytab, principal, {user.home}/krb5.keytab
+krb5Keytab=Kerberos keytab file
+krb5Keytab.desc=The path of the keytab file that is used to obtain the principal's secret key. If unspecified, the keytab from \
```

Comment:
If you specify the default location and name, then we need a complete list of OS .

Do we plan to support variable name?

Reply:
I checked in the JDK code, and the Login Module checks the Java System property `user.home` and `user.dir` regardless of OS.

Reply:
Yes, this is mostly copied/paraphrased from the JDK documentation, which is what we ultimately delegate to